interfaces/IBF-dashboard/src/app/services/map.service.ts

```diff
@@ -97,7 +97,11 @@ export class MapService {
       label: IbfLayerLabel[indicator.name],
       type: IbfLayerType.shape,
       description: 'loadAggregateLayer',
-      active: indicator.active,
+      active:
```

Comment:
@jannisvisser shouldn't the API send the correct value for active instead of changing this in the dashboard?

Reply:
@gulfaraz sure. you mean that also the trigger-boolean is checked in API-service? Feel free to adapt.